### ISSUE TITLE
User story: single-cell analysis

### ISSUE THREAD

**Issue by kieranrcampbell** [biothings/mygeneset.info#8]
We have multiple gene signatures for both cell types (e.g. CD3+ = T cell) or for pathway activation (e.g. KI67+ = proliferating). Some of these are canonical, others we find from papers or from doing our own differential expression. We'd like a common resource to easily store, display, discuss, and retrieve these, so we could say e.g. "Make sure all macrophages express the macrophage markers in the database".

**Comment by vincerubinetti** [comment]
When you say "discuss", did you imagine having a little chat section below the other information about a geneset? We could do this as an enhancement after MVP.

**Comment by kieranrcampbell** [comment]
Yeah that would be really nice -- you could imagine someone saying "I found gene X as a marker for cell type Y in this paper, but when I plot it on our dataset it doesn't appear to be" etc
